assignment1/isPermutation_test.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for isPermutation()."""
+
+from isPermutation import *
```

Comment:
Usually, one wants to avoid importing `*` because it may import more things than you originally intended when someone adds more things to the module. In this case, you really only need the `isPermutation` function.